src/main/webapp/src/app/ranking/ranking-list/ranking-list.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, OnInit } from '@angular/core';
 import { AppComponent } from 'src/app/app.component';
 import { LocationService } from '../../location.service';
 import { Location } from 'src/app/models/location';
```

Comment:
nit - we can remove location and location service now that we have the app component hidden gems